src/Bridge/Anthropic/PlatformFactory.php

```diff
@@ -6,12 +6,18 @@
 
 use PhpLlm\LlmChain\Platform;
 use Symfony\Component\HttpClient\EventSourceHttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
 
 final readonly class PlatformFactory
 {
-    public static function create(#[\SensitiveParameter] string $apiKey, string $version = '2023-06-01'): Platform
-    {
-        $responseHandler = new ModelHandler(new EventSourceHttpClient(), $apiKey, $version);
+    public static function create(
```

Comment:
As a separate method? What's the benefit?

Reply:
You can instantiate it and change it later, maybe because you want to inject a traceable http client later + you could return a clone to stay immutable, but just an idea and maybe not needed

Reply:
yes, that was basically the motivation here - to use the factory in the bundle but use the traceable one instantiated by the framework already